src/Glamour-Examples/GLMExamplesBrowser.class.st

```diff
@@ -17,7 +17,8 @@ GLMExamplesBrowser class >> menuExamplesOn: aBuilder [
 		parent: #Help;
 		order: 4.0;
 		help: 'Open Glamour Example Browser.';
-		action: [ self open ]
+		action: [ self open ].
+	aBuilder withSeparatorAfter	
```

Comment:
I wonder if it would not be better to group it with the help browser?

Just an idea, not something I really want.

Reply:
No - in most tools "About" (sometimes also called "Info") is the last item in the right most menu and often separated with a line before